components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1970,6 +1974,27 @@ do_default_build_config() {
     done
     chmod 755 "$pkg_prefix"/config
   fi
+  if [[ "${HAB_FEAT_INSTALL_HOOK:-}" = "true" && -d "$PLAN_CONTEXT/config_install" ]]; then
+    if [[ -z "${HAB_CONFIG_EXCLUDE:-}" ]]; then
+      # HAB_CONFIG_EXCLUDE not set, use defaults
+      config_exclude_exts=("*.sw?" "*~" "*.bak")
+    else
+      IFS=',' read -r -a config_exclude_exts <<< "$HAB_CONFIG_EXCLUDE"
+    fi
+    find_exclusions=""
```

Comment:
I copy/pasted this block from the same logic that deals with the `config` directory. So based on this, that logic is likely flawed as well. Its very likely that nothing uses that variable since I found no usage of it in our code base and it is undocumented as well.

I'm partly tempted to just remove the logic all together but maybe it is best to make that a separate PR. Will definitely make this right for now.